uint8[] LR_corner
uint8[] RR_corner
uint8[] LF_corner
uint8[] RF_corner
float32 resolution
uint32 width
uint32 height
bool left